app/src/main/java/com/gathering/android/event/home/FilterDialogFragment.kt

```diff
@@ -0,0 +1,50 @@
+package com.gathering.android.event.home
+
+import android.app.AlertDialog
+import android.app.Dialog
+import android.os.Bundle
+import android.view.KeyEvent
+import androidx.fragment.app.DialogFragment
+import com.gathering.android.R
+import com.gathering.android.databinding.FrgFilterDialogBinding
+
+class FilterDialogFragment(private val eventListViewModel: EventListViewModel) : DialogFragment() {
+
+    private lateinit var binding: FrgFilterDialogBinding
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+
+        binding = FrgFilterDialogBinding.inflate(layoutInflater)
+
+        val builder = AlertDialog.Builder(requireContext(), R.style.MyAlertDialogStyle)
+        builder.setView(binding.root)
+        builder.setTitle("Filter By")
+
+        builder.setNegativeButton("Cancel") { dialog, _ ->
```

Comment:
same here for "cancel"

Reply:
also lets not use AlertDialog negative button. add another button in your xml for dismissing the dialog, like the design in your ticket.